<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectWindow</class>
 <widget class="QMainWindow" name="ConnectWindow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>364</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>364</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FlareIRC - New Connection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Details</string>
    </property>
    <widget class="QSpinBox" name="spnPort">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>6667</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblPort">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblServer">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblChannel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Join Channel(s):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNick">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nick:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtServer">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNick">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtChannels">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOptions">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
